### ISSUE TITLE
False positive SC1008 when using bash inside conda

### ISSUE THREAD

**Issue by jamesbraza** [hadolint/hadolint#887]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

My `Dockerfile`:

```Dockerfile
FROM continuumio/miniconda3:4.12.0

# Copy files to /tmp to optimize Docker layer caching
COPY environment.yml /tmp/environment.yml

RUN conda env create -f /tmp/environment.yml
# Use conda environment as shell instead of default /bin/sh -c bash
ENTRYPOINT ["/opt/conda/condabin/conda", "run", "--no-capture-output", "-n", "tuna", "/bin/bash", "-c"]
# Use conda environment for all future RUN commands in this Dockerfile
SHELL ["/opt/conda/condabin/conda", "run", "--no-capture-output", "-n", "tuna", "/bin/bash", "-c"]

COPY requirements-qa.txt /tmp/requirements-qa.txt
RUN pip install --no-cache-dir --progress-bar off -r /tmp/requirements-qa.txt
```

Running `hadolint Dockerfile`:

```none
Dockerfile:13 SC1008 error: This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh. Add a 'shell' directive to specify.
```

I believe this is a valid shebang, it uses `/bin/bashbash` from the conda environment.  What do you think?

**Comment by jeremyh** [comment]
I'm seeing this too. Is there a reasonable workaround to tell it to treat the shell as normal bash?

**Comment by Bit0r** [comment]
I use fish, what should I do?
